cloud/filestore/libs/storage/tablet/tablet_ut_nodes.cpp

```diff
@@ -209,6 +220,46 @@ Y_UNIT_TEST_SUITE(TIndexTabletTest_Nodes)
         UNIT_ASSERT_VALUES_EQUAL(response->Record.GetSymLink(), "/ttt");
     }
 
+    Y_UNIT_TEST(ShouldCreateCharDevNode)
+    {
+        TTestEnv env;
+        env.CreateSubDomain("nfs");
+
+        ui32 nodeIdx = env.CreateNode("nfs");
+        ui64 tabletId = env.BootIndexTablet(nodeIdx);
+
+        TIndexTabletClient tablet(env.GetRuntime(), nodeIdx, tabletId);
+        tablet.InitSession("client", "session");
+
+        auto id1 = CreateNode(tablet, TCreateNodeArgs::CharDev(RootNodeId, "test1", 0, makedev(4, 1)));
+        tablet.AccessNode(id1);
+
+        auto stat = tablet.GetNodeAttr(id1)->Record.GetNode();
+        UNIT_ASSERT_VALUES_EQUAL(major(stat.GetDevId()), 4);
+        UNIT_ASSERT_VALUES_EQUAL(minor(stat.GetDevId()), 1);
+        UNIT_ASSERT_VALUES_EQUAL(static_cast<NProto::ENodeType>(stat.GetType()), NProto::E_CHARDEV_NODE);
```

Comment:
Let us do it without `Y_DECLARE_OUT_SPEC`? Just `UNIT_ASSERT_EQUAL` will suffice

Reply:
Sure, why not. Just thought it would be better looking and more consistent.